docs/semgrep-assistant/getting-started.md

```diff
@@ -119,6 +119,32 @@ Semgrep Assistant messages only appear in your PR comments for rules that are se
   ![ Policies modes](/img/semgrep-assistant-comment.png)
 * You have selected PR/MR comments in **Semgrep AppSec Platform > Settings > Deployment** in the **Code** section.
 
+### Add Memories (beta)
+
+Assistant Memories allow AppSec teams and developers to tailor Assistant's remediation guidance to their organization's standards and defaults on a per-project, per-rule basis. Whenever Assistant gives a suggested fix,  users can provide feedback by adding custom instructions.
+
+Memories are enabled by default for all Assistant users.
+
+To add a Memory:
+
+1. Identify the specific instance of **Assistant's suggested fix** that you want to modify. Click **Customize fix** to open an input box, and enter your preferred remediation approaches and secure defaults for the project. The suggestion you provide can be as general as "Use AWS Secrets Manager to manage secrets."
+   ![Assistant’s suggested fix for a hard-coded secret in the user’s code](/img/memories-1.png#md-width)
```

Comment:
```suggestion
   ![Assistant’s suggested fix for a hardcoded secret in the user’s code](/img/memories-1.png#md-width)
```
I am sorry to report that Google just wants to be different from MS and insists on hardcoded, not hard-coded. https://developers.google.com/style/word-list#hardcode